okhttp/src/jvmTest/java/okhttp3/MediaTypeTest.kt

```diff
@@ -0,0 +1,280 @@
+/*
+ * Copyright (C) 2013 Square, Inc.
+ * Copyright (C) 2011 The Guava Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3
+
+import java.nio.charset.StandardCharsets
+import java.util.Locale
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNull
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.MediaType.Companion.toMediaTypeOrNull
+import okhttp3.internal.platform.Platform.Companion.isAndroid
+
+/**
+ * Test MediaType API and parsing.
+ *
+ * This test includes tests from [Guava's](https://code.google.com/p/guava-libraries/)
+ * MediaTypeTest.
+ */
+open class MediaTypeTest {
+  protected open fun parse(string: String): MediaType = string.toMediaType()
```

Comment:
I think this should be `string.toMediaTypeOrNull()`, since it was

```
  protected MediaType parse(String string) {
    return MediaType.parse(string);
  }
```
while 

```
    @JvmStatic
    @JvmName("get")
    fun String.toMediaType(): MediaType {
```

and 

```
    @JvmStatic
    @JvmName("parse")
    fun String.toMediaTypeOrNull(): MediaType? {
```

Reply:
Yep, good call!

Reply:
(I switched it cause I had to get rid of `!!` in the source.)